cmd/svcat/binding/get_cmd_test.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package binding
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/test"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	svcatfake "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/fake"
+	"github.com/kubernetes-incubator/service-catalog/pkg/svcat"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+
+	_ "github.com/kubernetes-incubator/service-catalog/internal/test"
+)
+
+func TestGetCommand(t *testing.T) {
+	const namespace = "default"
+	testcases := []struct {
+		name         string
+		fakeBindings []string
+		bindingName  string
+		outputFormat string
+		expected     string
+		wantError    bool
+	}{
+		{
+			name:         "get non existing binding",
+			fakeBindings: []string{},
+			bindingName:  "mybinding",
+			expected:     "unable to get binding '" + namespace + ".mybinding': servicebindings.servicecatalog.k8s.io \"mybinding\" not found",
+			wantError:    true,
+		},
+		{
+			name:         "get all existing bindings with unknown output format",
+			fakeBindings: []string{"myfirstbinding", "mysecondbinding"},
+			bindingName:  "",
+			outputFormat: "unknown",
+			expected:     "",
+			wantError:    false,
+		},
+		{
+			name:         "get all existing bindings with json output format",
```

Comment:
These successful --output test cases are already covered by other tests, so we can remove the expectation on their full output.

Reply:
Removed them on latest commit